### DISCUSSION TITLE
The GCC 11.3.0 packages needed.

### DISCUSSION THREAD

**Discussion by TomaszHID** [msys2/MINGW-packages#20136] [Q&A]
Issue #20135 is closed, because of MSYS2 is a rolling based platform and the gcc-11.3.0 files were removed because packages are only kept for 1.5 years. My question is where I can find the following packages:
-	mingw-w64-x86_64-gcc-11.3.0-2-any.pkg.tar.zst
-	mingw-w64-x86_64-gcc-libs-11.3.0-2-any.pkg.tar.zst
-	mingw-w64-x86_64-gcc-libgfortran-11.3.0-2-any.pkg.tar.zst
-	mingw-w64-x86_64-gcc-ada-11.3.0-2-any.pkg.tar.zst
-	mingw-w64-x86_64-gcc-fortran-11.3.0-2-any.pkg.tar.zst
-	mingw-w64-x86_64-gcc-objc-11.3.0-2-any.pkg.tar.zst
-	mingw-w64-x86_64-libgccjit-11.3.0-2-any.pkg.tar.zst

I am working on the very old project and I need to downgrade GCC from 13.2.0 to 11.3.0.

**Comment by Biswa96** [comment]
Could you fix the project to use latest compiler toolchain?

**Reply by TomaszHID** [reply]
It is very complicated to do this.
Problem is that this project is very old and I am working on sustaining issues. It uses Free RTOS and I do not want to change/add anything "serious" in new release. Few fixes are needed only.
Currently I have a problem with linkage stage and I have the followong errors:
undefined reference to `__exidx_end'
undefined reference to `__exidx_start'
undefined reference to `_exit'
undefined reference to `_kill'
undefined reference to `_getpid'
undefined reference to `_close'
undefined reference to `_lseek'
undefined reference to `_read'
undefined reference to `_sbrk'
undefined reference to `_write'

I believe that GCC 11.3.0 will fix my problems.

**Comment by smalltalkman** [comment]
Maybe [winlibs](https://winlibs.com/) has the gcc version you need.

**Reply by TomaszHID** [reply]
Thank you. I will try this.
